paragon/_overrides.scss

```diff
@@ -140,9 +140,27 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-p {
-  a[href]:not(.btn) {
-    text-decoration: underline;
+p > a[href]:not(.btn),
```

Comment:
Isn't this the style for links IN paragraphs?  Don't we want that to still have an underline?

Reply:
The line is generated with the pseudo elements `:after`